src/server/index.ts

```diff
@@ -40,6 +40,10 @@ const build = () => {
     } catch (error) {
       server.log.error(error)
 
+      if (error?.name === 'SyntaxError') {
```

Comment:
What is your opinion about [`error handlers`](https://www.fastify.io/docs/v3.8.x/Server/#seterrorhandler)? It could be a more elegant solution and adds more scalability. It seems easy to implement aswell.

https://dev.to/eomm/fastify-error-handlers-53ol

Reply:
Changed! Thanks for the tip :)